node/lib/ThingAccessor.ts

```diff
@@ -1,53 +1,53 @@
 
-import { DeviceInterface } from "./DeviceInterface";
-import { ITranslator } from "./ITranslator";
+import { ThingSchema } from "./ThingSchema";
+import { IThingTranslator } from "./IThingTranslator";
 
 import { EventEmitter } from "events";
 
 /**
- * Provides reflection-style access to device properties and methods via device interfaces.
+ * Provides reflection-style access to thing properties and methods via translators.
  */
-export class DeviceAccessor {
+export class ThingAccessor {
```

Comment:
Sure, I didn't love the name either (before or after the change), but I didn't think of anything better.